packages/trace-viewer/src/ui/workbenchLoader.css

```diff
@@ -100,6 +100,13 @@ body.dark-mode .drop-target {
 .workbench-loader .logo {
   font-size: 20px;
   margin-left: 16px;
+  display: flex;
+}
+
+.workbench-loader .logo img {
+  height: 32px;
+  align-self: center;
+  pointer-events: none;
```

Comment:
why no pointer events? also, why no width?

Reply:
no pointer events for cosmetic UX reasons, since then immediately the trace upload would get triggered.